server/middleware/setUpWebSecurity.ts

```diff
@@ -25,6 +25,7 @@ export default function setUpWebSecurity(): Router {
   const styleSrc = ["'self'", (_req: Request, res: Response) => `'nonce-${res.locals.cspNonce}'`]
   const imgSrc = ["'self'", 'data:']
   const fontSrc = ["'self'"]
+  const connectSrc = ["'self'", '*.applicationinsights.azure.com']
```

Comment:
We need to configure the web security component (`helmet` 🤣 🤐 ) to allow connections outbound to the app insights domain